tests/foreman/cli/test_capsule.py

```diff
@@ -382,3 +386,25 @@ def test_positive_enable_bmc():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.skip_if_not_set('fake_capsules')
+@pytest.mark.tier1
+def test_positive_search_nondefault_proxy(request, target_sat):
+    """Search non-default proxy  with the home proxy
+
+    :id: caf51662-6b4e-11ed-baba-2b9d7b368002
+
+    :expectedresults: Non-default proxy can be searched
+
+    :BZ: 2077824
+
+    :customerscenario: true
+    """
+    proxy = _make_proxy(request, target_sat, options={'name': gen_string('alpha')})
+    path = target_sat.api.Capsule().path() + '?search=id+%21%3D+1'
+    response = client.get(path, auth=get_credentials(), verify=False)
+    assert response.status_code == 200
+    assert len(json.loads(response.text)['results']) == 1
+    assert json.loads(response.text)['results'][0]['name'] == proxy.name
+    assert json.loads(response.text)['results'][0]['url'] == proxy.url
```

Comment:
What about using nailgun as we do it in the other modules?
```suggestion
    capsule_search = target_sat.api.Capsule().search(query={'search': 'id != 1'})
    assert len(capsule_search) == 1
    assert capsule_search[0].name == proxy.name
    assert capsule_search[0].url == proxy.url
```

Reply:
@ogajduse it was not working for me earlier somehow, its working now, updated